commercetools-sdk-base/pom.xml

```diff
@@ -74,16 +80,132 @@
             <groupId>com.fasterxml.jackson.module</groupId>
             <artifactId>jackson-module-parameter-names</artifactId>
             <version>${jackson.version}</version>
+            <exclusions>
```

Comment:
Would be nice if we could add these exclusions to our parent pom, just to avoid potential duplication.